uberfire-security/uberfire-security-management/uberfire-security-management-wildfly/uberfire-security-management-wildfly10/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@manstis not following you sorry... :/ Prolog is fine isn't it? I was not sure if the whole file was really necessary....  ( sorry it's late and I'm sometimes confused... only asking :)